integrations/cloudbuild/chef.yaml

```diff
@@ -29,7 +29,7 @@ steps:
       args:
           - >-
               perl -i -pe 's/^gdbgui==/# gdbgui==/' /opt/espressif/esp-idf/requirements.txt &&
-              ./examples/chef/chef.py --build_all --build_exclude "noip|temperaturecontrolledcabinet"
+              ./examples/chef/chef.py --build_all --build_exclude "noip|temperaturecontrolledcabinet|ed3b19ec55"
```

Comment:
`ed3b...` is not that friendly for users to read. Could we rename it as ICD maybe or use some different description?

Could we explain in comments why these are excluded?

Reply:
@andy31415 The icd is an cluster instead of a device type of an endpoint that we put it in the Chef's filename. So I just add some comments in the description. PTAL.

Reply:
The description comment doesn't help understand. I don't understand why we are keeping using the weird hashes in the device names. Nobody has actually done anything to keep track of those.

Reply:
It should not say `ed3b19ec55` ... it should look like `icd` or something like that. An `I want to exclude 12acb3d` statement for example is not human reviewable and maintainable. It has no meaning as a number/hex code. Please use human readable strings.

Reply:
You can change the naming conventions if need be and mark things for ICD if that is important. However do not use hex "signatures" to identify things.

Reply:
Got it. I'll follow the `noip` rule to rename it as `icd_rootnode_contactsensor_ed3b19ec55`